static/js/notifications.js

```diff
@@ -635,17 +643,18 @@ export function notify_local_mixes(messages, need_user_to_scroll) {
 
         const link_msg_id = message.id;
         const link_class = "compose_notification_narrow_by_topic";
-        const link_text = $t(
-            {defaultMessage: "Narrow to {message_recipient}"},
-            {message_recipient: get_message_header(message)},
-        );
+        const default_message = "Narrow to";
```

Comment:
I think `$t` was used here for translation purposes, so I think we should still stick with using it, plus I don't think your implementation requires you to remove it anyways, you can just let that line be and use it in the `compose_notification` template as it was used before with your additional changes.

Reply:
Now I have used $t along with my changes.